Donation-platform/src/main/java/com/google/objectify/ObjectifyBootstrapper.java

```diff
@@ -0,0 +1,18 @@
+package com.google.objectify;
+
+import com.google.model.Charity;
+import com.google.model.Tag;
+import com.google.model.Users;
+import com.googlecode.objectify.ObjectifyService;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.servlet.annotation.WebListener;
+
+@WebListener
+public class ObjectifyBootstrapper implements ServletContextListener {
```

Comment:
what does this class do, please add documentation

Reply:
Resolved